### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by juliohm**
Update and simplify README to reflect latest features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,21 @@
 # GslibIO.jl
 
-Utilities to read/write *extended* [GSLIB](http://www.gslib.com/gslib_help/format.html) files in Julia.
+Utilities to read/write *extended* and *legacy* [GSLIB](http://www.gslib.com/gslib_help/format.html) files in Julia.
```
Comment (exepulveda) [inline README.md:3]: Users may confuse that the link provided includes the extended format, but only includes the legacy one.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,21 @@
...
+## Overview
 
-The GSLIB file format was introduced a long time ago for storing regular grids in text files that are easy to read. The format specification is incomplete mainly because:
+The GSLIB file format was introduced a long time ago for storing spatial data over regular grids
+and point sets in text files that are easy to read. The format specification is incomplete mainly because:
 
 1. it doesn't contain the size of the grid (i.e. `(nx, ny, nz)`)
 2. it doesn't specify the origin and spacing (i.e. `(ox, oy, oz)`, `(sx, sy, sz)`)
 3. it doesn't specify the special symbol for inactive cells (e.g. `-999`)
+4. it doesn't specify which variable names are coordinates
```
Comment (exepulveda) [inline README.md:16]: I would start the sentences in 1-3 with "for the grid case", because for point sets are not used

**Review by exepulveda — COMMENTED** [review]
It is useful for users to have the examples for load/save legacy, but I see that you removed that section.

File: README.md (hunk 1 above)
Comment (juliohm) [reply README.md:3]: That is a good point, but we don't have a link to the extended one. So what can we do? Any suggestion to rephrase the sentence? I think it should be fine as most users will already be familiar with GSLIB anyways, so they will know what this is about.

File: README.md (hunk 2 above)
Comment (juliohm) [reply README.md:16]: Nice. I will split 1-3 from 4 tomorrow morning.

File: README.md (hunk 1 above)
Comment (exepulveda) [reply README.md]: I understand that extended format is the proposed format of this package. I am not aware of any formal extended GSLIB format (except this one). In that sense I would suggest something like this:
```
Extend the GSLIB format [GSLIB](http://www.gslib.com/gslib_help/format.html) and provide utilities to read/write files in Julia. For backward compatibility it also provides utilities to read/write *legacy* format.
```

**Review by exepulveda — CHANGES_REQUESTED** [review]
I think this version is good enough. Just the header need some adjustment as I pointed out in my previous review. See my comments.

File: README.md (hunk 1 above)
Comment (juliohm) [reply README.md]: I think the proposed sentence is more confusing than the current one. The current one is to the point: it says what the package provides right away. I understand we extend the format, but that is not the main contribution of the software. Let's keep the current sentence as is and review it in the future if needed. It should be clear enough. 👍🏽

**Comment by juliohm** [comment]
Thanks for the review! Let's keep the first sentence as is for now because the other suggestions are more confusing. If in the future we figure out something else, we can always change. I think there are other priorities to fix 👍🏽

**Pull request merged**